conf/local.conf.sample

```diff
@@ -9,9 +9,6 @@ BB_NUMBER_THREADS ?= "${@oe.utils.cpu_count()}"
 PARALLEL_MAKE ?= "-j ${@oe.utils.cpu_count()}"
 
 PACKAGE_CLASSES ?= "package_rpm"
-EXTRA_IMAGE_FEATURES_append = " debug-tweaks "
```

Comment:
So where did those go, into the image? I always like some comment on this in the commit message so it's clear if this got just removed and for what reason and if just moved where to.